source/zh-cn/docs/commands.md

```diff
@@ -77,23 +77,23 @@ $ hexo server
 
 启动服务器。 默认情况下，访问网址为： `http://localhost:4000/`。
 
-| 选项               | 描述                                     |
-| ---------------- | -------------------------------------- |
-| `-p`, `--port`   | 重设端口                                   |
-| `-s`, `--static` | 只使用静态文件                                |
-| `-l`, `--log`    | Enable logger. Override logger format. |
+| 选项               | 描述             |
+| ---------------- | -------------- |
+| `-p`, `--port`   | 重设端口           |
+| `-s`, `--static` | 只使用静态文件        |
+| `-l`, `--log`    | 启用日志。 覆盖记录器格式。 |
 
 ## deploy
 
 ```bash
 $ hexo deploy
 ```
 
-Deploys your website.
+部署你的网站.
 
-| Option             | 描述                         |
-| ------------------ | -------------------------- |
-| `-g`, `--generate` | Generate before deployment |
+| 选项                 | 描述     |
+| ------------------ | ------ |
+| `-g`, `--generate` | 在部署前生成 |
```

Comment:
Machine Translation. Should we accept this?

Reply:
No. We certainly need better translations.